snapshot-labs/sx-monorepo
# fix: add fallback icon to preview link icon

### Summary



Closes: https://github.com/snapshot-labs/workflow/issues/246

This PR adds a fallback icon when the icon in the preview link is not resolving

### How to test

1. Go to Editor 
2. Put a valid discussion link
3. It should show the icon
4. (Not sure about how to test invalid icon) Try putting some url without icon
5. It should show the fallback (zap) icon
